doc/index.md

```diff
@@ -0,0 +1,59 @@
+# Welcome to the PyPSA-Earth-Status documentation!
+
+[![Test workflows](https://github.com/pypsa-meets-earth/pypsa-earth-status/actions/workflows/test.yaml/badge.svg)](https://github.com/pypsa-meets-earth/pypsa-earth/actions/workflows/test.yaml)
+![Size](https://img.shields.io/github/repo-size/pypsa-meets-earth/pypsa-earth-status)
+[![License: AGPL v3](https://img.shields.io/badge/License-AGPLv3-blue.svg)](https://www.gnu.org/licenses/agpl-3.0)
+[![Discord](https://img.shields.io/discord/911692131440148490?logo=discord)](https://discord.gg/AnuJBk23FU)
+[![Google Drive](https://img.shields.io/badge/Google%20Drive-4285F4?style=flat&logo=googledrive&logoColor=white)](https://drive.google.com/drive/folders/13Z8Y9zgsh5IZaDNkkRyo1wkoMgbdUxT5?usp=sharing)
+
+*Motivation*. Validating energy system models is crucial for ensuring their accuracy and reliability. However, this process can be time-consuming and complex, often requiring extensive manual effort to compare model outputs with real-world data. PyPSA-Earth-Status addresses this challenge by providing an automated framework for validating energy networks against established energy system statistics, also maintaining up-to-date statistics on the accuracy of PyPSA-Earth.
+
+
+## Documentation
+
+<div class="grid cards" markdown>
+
+-   :material-clock-fast:{ .lg .middle } **Getting started**
+
+    ---
+    
+    [:octicons-arrow-right-24: Installation](home/installation.md)
+
+    [:octicons-arrow-right-24: Quick Start](home/quick-start.md)
+
+-   :material-view-list:{ .lg .middle } **User guide**
+
+    ---
+    
+    [:octicons-arrow-right-24: Features](user-guide/user-guide.md)
```

Comment:
Do we really need all this stuff? It adds a number of the elements which seem not really well playing together (see https://pypsa-earth-status.readthedocs.io/en/latest/#documentation) and it'll take quite some effort to polish it